go/mysql/auth_server_static.go

```diff
@@ -116,13 +113,12 @@ func RegisterAuthServerStaticFromParams(file, str string) {
 	}
 	authServerStatic.installSignalHandlers()
 
-	authServerStatic.LastAuthRotation = time.Now()
-
 	// And register the server.
 	RegisterAuthServerImpl("static", authServerStatic)
 }
 
 func (a *AuthServerStatic) loadConfigFromParams(file, str string) {
+	// a.Entries is accessed within this function, so we lock here
```

Comment:
I think you should move the lock _after_ reading the file though rather than holding it during the i/o processing.